docs/install/installing-istio.md

```diff
@@ -261,6 +261,54 @@ kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_
 **Note:** This method is only for development purposes. The production readiness of the above 
 installation method is not ensured. For a production-ready installation, see the `helm` installation method above.
 
+After you install the cluster local gateway, your service and deployment for the local gateway are both named `cluster-local-gateway`.
+You do not need to update the `config-istio` configmap under `knative-serving` namespace, because Knative Serving can use
```

Comment:
"under the `knative-serving` namespace"

"the local gateway `cluster-local-gateway` by default"